### PR TITLE
Removing false information: Dawnguard Epilogues

### PR DESCRIPTION
**Pull request by SkyLover264**
As the mod author of Dawnguard and Clan Volkihar Epilogues (Dawnguard&VolkiharArtifactsQuests.esp), I can tell you with certainty that Legacy of the Dragonborn, and my mod, are not incompatible. I've used them side-by-side since I made it, even with V5 of Legacy, and they work together without a problem.

Likewise, the 4 itms are false reports, and are needed for the mod to work.

### CODE REVIEW

**Comment by pStyl3** [comment]
The incompatible message was issued in this issue: https://github.com/loot/skyrimse/issues/927
The information comes from this [article](https://www.nexusmods.com/skyrimspecialedition/articles/1494) from the Legacy of the Dragonborn SSE Nexus page.
```
Dawnguard/Clan Volkihar Epilogues - Legacy already has a solution built into it to provide all of the path specific relics from the Dawnguard questlines. Do not install this mod.
```

**Comment by SkyLover264** [comment]
I originally responded to the Legacy team's assessment on my own mod page. I'll repost it here:

"Over the past year, I've seen some misinformation spreading that this mod is not compatible with Legacy of the Dragonborn. I would like to shoot that down now. Aside from some duplicate items, this mod and Legacy do NOT interfere with each other in any way, shape, or form. In fact, my mod provides some things that Legacy does not. I've decided to address this here as I am sick and tired of correcting every post I see claiming this. Henceforth, I ask that you not continue to spread this misinformation any further."

**Review by themayor897 — CHANGES_REQUESTED** [review]
It's irresponsible to completely remove the entry from the masterlist rather than edit it. Also, I think it's wise to wait for the LOTD team to weigh in on this before committing any changes.

File: masterlist.yaml [hunk 1]
```diff
@@ -9719,15 +9719,6 @@ plugins:
       - crc: 0x53109251
         util: 'SSEEdit v4.0.2f'
 
-  - name: 'Dawnguard&VolkiharArtifactsQuests.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/12098' ]
-    inc: [ *LotD_v5 ]
```
Comment (themayor897) [inline masterlist.yaml:9724]: msg: [ *doNotClean ]

File: masterlist.yaml [hunk 2]
```diff
@@ -9719,15 +9719,6 @@ plugins:
...
-    dirty:
```
Comment (themayor897) [inline masterlist.yaml:9725]: if the donotclean message is added lines 9725-end should be removed.

**Review by themayor897 — CHANGES_REQUESTED** [review]

**Comment by MacSplody** [comment]
There's a compromise which can be reached regarding the views on compatibility.

I added a message in https://github.com/loot/skyrimse/commit/e464a908dbd10f409b6fb651374dccbff3bcbd35 which could work here. It would need to be added to the `LegacyoftheDragonborn.esm` metadata.

**Comment by SkyLover264** [comment]
> It's irresponsible to completely remove the entry from the masterlist rather than edit it. Also, I think it's wise to wait for the LOTD team to weigh in on this before committing any changes.

I don't think their word would be trustworthy here. They hate me.

File: masterlist.yaml [hunk 3]
```diff
@@ -9718,16 +9718,11 @@ plugins:
       # version:  1.0r
       - crc: 0x53109251
         util: 'SSEEdit v4.0.2f'
-
+
```
Comment (MacSplody) [inline masterlist.yaml]: No tabs should be used, only spaces.
```suggestion

```

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
@SkyLover264 Thanks.

**Pull request merged**